src/ucp/core/ucp_am.c

```diff
@@ -951,6 +955,11 @@ UCS_PROFILE_FUNC(ucs_status_ptr_t, ucp_am_send_nbx,
 
     UCP_WORKER_THREAD_CS_ENTER_CONDITIONAL(worker);
 
+    if (id & 0xffff0000) {
+        ret = UCS_STATUS_PTR(UCS_ERR_INVALID_PARAM);
```

Comment:
I guess we need to log an error with `ucs_error` macro

Reply:
I see that using `ucs_error` fails the test automatically (I wanted to specifically check for a failure when the id is too large). @iyastreb  What is preferable? issuing a ucs_error message or testing this case? or is there another solution?

Reply:
for test to ignore error, you could use `scoped_log_handler slh(hide_errors_logger);`

Reply:
maybe use UINT16_MAX